### ISSUE TITLE
If I activate my extension, I am redirected to list of contributors page

### ISSUE THREAD

**Issue by MaartenLMEM** [dis-moi/extension#821]
**Describe the feature/improvement**
If I activate my extension, I am redirected to list of contributors page

**Target user**
- Users that did not open his computer since a long time
- Users who s chrome account had to synchonize
- Other reasons that desactivate a chrome extension

**Fill in the following areas if you have the time...**
Here is wh'at happening currently : nothing : https://drive.google.com/file/d/1nMBbjXbvoHDdCja7LOXJdr9opgOCAaf8/view

**Additional context**
Add any other context about the feature/improvement here. 

**User stories**
How will the user experience this feature/improvement?
1. The user can click a button in [e.g., the extension]
2. User then sees a screen that says [something]
3. ...

**Technical implementation notes**
Any ideas you have about how this could be implemented.

**Comment by christpet** [comment]
Womp womp. This is a great quick win to implement. @lutangar how big does this seem to you? Xs - s - m - lg ?

**Comment by lutangar** [comment]
Yes, this is totally doable, but will require yet another permission: `management`.
> - https://developer.chrome.com/docs/extensions/reference/management/#event-onEnabled
> - https://developer.mozilla.org/fr/docs/Mozilla/Add-ons/WebExtensions/API/management/onEnabled

As before, I would recommend to make permissions `optionnal` as much as possible and I think this is a good use case.
I'll set complexity label.

**Comment by JalilArfaoui** [comment]
I totally agree that extension should work fine if the user rejects the management authorization, but it’s still a nice feature

**Comment by christpet** [comment]
Because this requires a permission authorization @lutangar we will wait to implement it until we can put it in a batch of other improvements that also require permissions. 

I'll go through the list of permissions we might need in the future and we can start adding small features like this to it! I'll add the new "needs-permissions" label to this so we remember.

**Comment by MaartenLMEM** [comment]
@lutangar @JalilArfaoui Entre temps j'ai expliqué à Christian un point important que vous connaissez : plus on demande les autorisations tardivement avec Chrome, plus on va perdre d'utilisateur 
car Chrome fait apparaitre chaque petite demande de nouvelle autorisation comme un demande complète comprenant 'accéder à toutes les données des sites web que vous visitez".

Conclusion : A mon sens il faut passer cette feature au plus vite.

**Comment by JalilArfaoui** [comment]
@MaartenLMEM Unfortunately, the `management` permission does trigger a warning message to authorize `Manage your apps, extensions, and themes`

See : https://developer.chrome.com/docs/extensions/mv2/permission_warnings/#permissions_with_warnings
